src/scripts.js

```diff
@@ -131,8 +140,14 @@ $(document).ready(function () {
         }]
       }
     }
-  });
+  })
+};
 
+fetch('https://fe-apps.herokuapp.com/api/v1/fitlit/1908/hydration/hydrationData')
+                                          .then(response => response.json())
+                                          .then(data => hydrationData = data.hydrationData)
+                                          .then(() => hydration = new Hydration(hydrationData, user.id))
+                                          .then(() => hydrationDOM())
```

Comment:
I like this solution to the issue with the API being too slow - now the DOM elements will always render last.